engine-nextgen/src/main/kotlin/graphql/nadel/NextgenEngine.kt

```diff
@@ -35,6 +37,7 @@ import java.util.concurrent.CompletableFuture
 
 class NextgenEngine @JvmOverloads constructor(nadel: Nadel, transforms: List<NadelTransform<out Any>> = emptyList()) : NadelExecutionEngine {
     private val services: Map<String, Service> = nadel.services.strictAssociateBy { it.name }
+    private val serviceExecutionHooks: ServiceExecutionHooks = nadel.serviceExecutionHooks
```

Comment:
uses the existing `ServiceExecutionHooks` instance that is already being passed in `nadel`.